### DISCUSSION TITLE
Using whiteboards

### DISCUSSION THREAD

**Discussion by michael-dm** [notaapp/talk#159] [Feature Requests]
Nota is currently the best Markdown editor app, no doubt !
I just feel it lacks ways to see notes visually, make connections, have a higher view...

There is of course a "graph view", which would somewhat helps (but is far from perfect).
But there is also an other very interesting way, that I discovered on https://heptabase.com. Whiteboards !

https://user-images.githubusercontent.com/26444186/154662889-bf33efa0-082d-4b4a-89c0-e5f3527b2758.mp4

Any file can be imported in any whiteboard and you can make connections between them.
What do you think ?

**Comment by astoilkov** [comment]
> Nota is currently the best Markdown editor app, no doubt !

Wow. Thanks for the kind words.

> I just feel it lacks ways to see notes visually, make connections, have a higher view...

We definitely agree. Nota is great at editing and creating content. However, we want to make it great for exploration too. We also want users to detect patterns in their notes and notice what is most important to them. We already have 3 private experiments that are exploring better ways to explore your notes.

> There is of course a "graph view", which would somewhat helps (but is far from perfect).

We agree that "graph view" is far from perfect. In order to distinguish hype from real value creation we will need some time. Imagine we do some wild experiment that doesn't turn out to be a great idea — we'll be stuck with it and removing it will probably upset a lot of people (we also can't do this every month). There are a lot of apps that probably see a drop in their usage because they started doing the wrong things while needing to support not so useful concepts.

> Any file can be imported in any whiteboard and you can make connections between them.
What do you think ?

Whiteboards are a new concept for us. We are yet to explore it more deeply. Currently, I can't say more that it looks like a very interesting idea that is worth exploring.

Conclusion: Thanks a lot for the idea.

**Comment by michael-dm** [comment]
That's a great thing to hear ! Good luck with your experiments.
You seem like a very wise team with a promising future 👍

**Reply by O-Schell** [reply]
Et donc concrètement, la suggestion que je veux faire pour un de mes éditeurs Markdown favoris se trouve être déjà faite par le seul et unique Micode...
<br>
- ****Bucketlist des truc improbable***
- [x] Suggestion github déjà faite par mon youtubeur tech favoris (déso pas déso hardisk )
